guides/plugins/plugins/framework/rule/add-custom-rules.md

```diff
@@ -161,6 +173,39 @@ import './decorator/rule-condition-service-decoration';
 It may be possible that rules, with your newly created condition, aren't selectable in some places inside the Administration — for example, inside the promotion module. That is because rules are "context-aware". To learn more about that feature [click here](#context-awareness)
 :::
 
+#### Creating a new group in the administration
+The rule will now be added to the list of rules in the admin. It might be useful to create a new group for your rules. We can create a new group by using the `upsertGroup` method of the [RuleConditionService](https://github.com/shopware/shopware/blob/v6.3.4.1/src/Administration/Resources/app/administration/src/app/service/rule-condition.service.js).
```

Comment:
@jasperP98 any reason you used the 6.3 docs here? It currently is flagged as 6.5

The files were changes from js to ts by then. Function is still available tho.

https://github.com/shopware/shopware/blob/v6.5.8.7/src/Administration/Resources/app/administration/src/app/service/rule-condition.service.ts

Same goes for 6.6 so I am currently thinking to go with 6.6

Reply:
@Isengo1989 no specific reason actually, i will change that ASAP

Reply:
@jasperP98 thx. Could you please also use // [!code focus] instead of highlight? I double checked locally and highlight is not supported yet. Will have to create an internal PR for that first.

Reply:
![image](https://github.com/shopware/docs/assets/8600299/80504190-2a8f-45a8-a53c-24b8d93bc542)

Example

Reply:
@Isengo1989 I updated the PR. It looks like focus does not work either.

Reply:
@jasperP98 thx, for me it worked (see above screenshot). Should be merged next week, the colleague who tagged it 6.5 is currently on vacation. Just making sure he had no direct issue with it being merged to 6.6 @bojanrajh fyi

Reply:
@Isengo1989 okay great thanksi